pkg/scheduler/service/cleaner.go

```diff
@@ -53,18 +53,30 @@ func (c *cleaner) Run(ctx context.Context, transition *ClusterStatusTransition,
 	}
 }
 
-func (c *cleaner) purgeReconciliations(transition *ClusterStatusTransition, config *CleanerConfig) {
-
+func (c *cleaner) purgeEntities(transition *ClusterStatusTransition, config *CleanerConfig) {
 	c.logger.Infof("%s Process started", CleanerPrefix)
 
-	if config.KeepLatestEntitiesCount > 0 {
-		c.logger.Infof("%s Cleaner will remove unnecessary entities", CleanerPrefix)
+	// delete reconciliations
+	if config.keepLatestEntitiesCount() > 0 {
+		c.logger.Infof("%s Cleaner will remove unnecessary reconciliations", CleanerPrefix)
 		c.purgeReconciliationsNew(transition, config)
 	} else {
-		c.logger.Infof("%s Cleaner will remove entities older than %s", CleanerPrefix, config.PurgeEntitiesOlderThan.String())
+		c.logger.Infof("%s Cleaner will remove reconciliations older than %s", CleanerPrefix, config.PurgeEntitiesOlderThan.String())
 		c.purgeReconciliationsOld(transition, config)
 	}
 
+	// delete cluster entities
+	clusterInventoryCleanupDays := 20 // days default
```

Comment:
I think that the default should not be set here, but should instead come from configuration/flags definition

Reply:
Makes sense, let's take a decision on this in a follow-up